hack/tools/go.work

```diff
@@ -0,0 +1,7 @@
+go 1.19
```

Comment:
Why should the sub-package need its own go.work + go.work.sum?

Reply:
:laughing: let's keep the conversation over on the other PR where it's more visible